storage/collections/operator_storage.go

```diff
@@ -0,0 +1,113 @@
+package collections
```

Comment:
@nivBlox this whole thing feels as a duplicate for validator collection

Reply:
as we discussed, operator managing only operator's keys and validator accordingly

Reply:
operator will later be moved to its own package.